internal/statemachine/common_states.go

```diff
@@ -448,11 +448,26 @@ func (stateMachine *StateMachine) makeDisk() error {
 	return nil
 }
 
+// VolumeMaxSize get the max size of a volume from its structures
+// TODO remove once https://github.com/canonical/snapd/pull/14561 is merged
+func VolumeMaxSize(v *gadget.Volume) quantity.Size {
```

Comment:
similarly to the snapd PR, I'd suggest slightly different name, eg. `volumeDefaultSize()`

Reply:
This function is here until snapd is fixed, then I will remove it and use the snapd Volume method.